app/helpers/sheet/group/nav_left.rb

```diff
@@ -31,8 +31,31 @@ def render
 
       private
 
+      def sort_list_by_name(objects)
+        children_of = {}
+        objects.each do |o|
+          children_of[o.parent_id] ||= []
+          children_of[o.parent_id] << o
+        end
+
+        children_of.each_value do |children|
+          children.sort_by! { |c| c.short_name || c.name }
+        end
+
+        results = []
+        recombine_lists(results, children_of, objects.first.parent_id)
+        results
+      end
+
+      def recombine_lists(results, children_of, parent_id)
+        children_of[parent_id]&.each do |o|
+          results << o
+          recombine_lists(results, children_of, o.id)
+        end
+      end
+
       def groups
-        @groups ||= entry.groups_in_same_layer.without_deleted.order(:lft).to_a
+        @groups ||= sort_list_by_name entry.groups_in_same_layer.without_deleted.order(:lft)
```

Comment:
@mtnstar This implements the sorting for the groups, but not the separately listed sections. These seem to be sorted when changed, but behave a bit strange.

Reply:
Fixed with https://github.com/hitobito/hitobito_sac_cas/pull/621